### DISCUSSION TITLE
Compile for chrome out there thats already compiled?

### DISCUSSION THREAD

**Discussion by SkullexGH** [Leather128/LeathersFunkinEngine#122] [General]
Hello!

i was wondering if someone had already compiled the game into html5 if thats possible, if so, id like the files to the compile (so i can play it)

Thanks!

Edit: im not uploading this crap, i want it for myself, not like in a stealing way though

**Comment by Leather128** [comment]
uh you just need the source code? i mean it's right there, and as of the most recent github action it seems like html5 can build, if you want however, you can download that html5 build from [here](https://github.com/Leather128/LeatherEngine/suites/4832158212/artifacts/136659595)

**Reply by SkullexGH** [reply]
i just dont know how to compile-
thats the problem lol

**Reply by SkullexGH** [reply]
but thanks :)

**Reply by Leather128** [reply]
read build.md
